.github/workflows/update-images.yaml

```diff
@@ -109,11 +116,11 @@ jobs:
     - uses: actions/checkout@v2
       with:
           fetch-depth: 0
-          
+
     - id: modified
       name: Check for modified paths
       run: |
-        PATHS=(".github/workflows/update-images.yaml" "kctf-operator/*" "kctf-operator/build/**" "kctf-operator/cmd/**" "kctf-operator/pkg/**" "kctf-operator/version/**")
```

Comment:
do you remember why we didn't have "**" here before? Maybe we didn't want to trigger the workflow if some files changed?

Reply:
not sure, looking at [the code](https://github.com/google/kctf/tree/3ef69474bc16039f8fcccb5f4d3341be1e8a5951/kctf-operator) it was only ignoring `deploy` and `samples`, but I think we rarely changed `samples` and `deploy`. we can make it more granular in the future if this causes delays

Reply:
yeah, sounds good. I just wondered if there's a chance this leads to an endless loop.
* detect changes
* rebuild
* update operator.yaml with new image tag
* detect changes
* ...

Reply:
hmm I don't think so, the steps can only run once and they wont re-run on a robot-generated change :D